zerver/lib/upload/s3.py

```diff
@@ -102,6 +103,7 @@ def upload_image_to_s3(
         ContentType=content_type,
         ContentDisposition=content_disposition,
         StorageClass=storage_class,
+        CacheControl=cache_control,
```

Comment:
Does passing  `CacheControl=""` into `key.put` do nothing, or upload an empty-string header? Just wondering if we need to make passing the parameter optional.

Reply:
It does result in responses with an empty `CacheControl:` header, though I can't imagine that would cause problems.

Making the parameter optional requires a `type: ignore` because it doesn't support passing the parameter as None, and mypy doesn't like unpacking an extra set of options as a dict.

Reply:
OK that's probably fine.